tests/integration/requests_tests.py

```diff
@@ -170,179 +168,3 @@ def test_external_client_doesnt_send_headers(http_server):
         assert "X-Parent" not in http_server.requests[0].headers
         assert "X-Span" not in http_server.requests[0].headers
         assert "X-Edge-Request" not in http_server.requests[0].headers
-
-
-@pytest.mark.parametrize("client_cls", [InternalRequestsClient, ExternalRequestsClient])
```

Comment:
Are there any tests to replace the stuff that was deleted here?

Reply:
The happy paths are covered by existing unit tests now (caught a few misteaks like that), noodling on how to test errors and exceptions :thinking:

Reply:
the problem is we don't actually know how the client behave
The behavior we expect is tested. It doesn't mean it's the real behavior. Previous maintainer of the code made the assumption that `baseplate.smth.Error` would be raised and the code/test are structured around that.
Real world errors might be diff.